react-components/src/architecture/base/concreteCommands/pointCloud/PointCloudDividerCommand.test.ts

```diff
@@ -0,0 +1,19 @@
+import { PointCloudDividerCommand } from './PointCloudDividerCommand';
+import { createFullRenderTargetMock } from '../../../../../tests/tests-utilities/fixtures/createFullRenderTargetMock';
```

Comment:
Should be imported as
```suggestion
import { createFullRenderTargetMock } from '#test-utils/fixtures/createFullRenderTargetMock';
```